controllers/activemqartemis_reconciler.go

```diff
@@ -2173,6 +2206,96 @@ func brokerPropertiesData(props []string) map[string]string {
 	return map[string]string{brokerPropertiesName: buf.String()}
 }
 
+func (reconciler *ActiveMQArtemisReconcilerImpl) addConfigMapForAuthProperties(customResource *brokerv1beta1.ActiveMQArtemis) string {
+
+	configMapName := types.NamespacedName{
+		Namespace: customResource.Namespace,
+		Name:      customResource.Name + "-jaas-mgmt",
+	}
+
+	var desired *corev1.ConfigMap
+
+	obj := reconciler.cloneOfDeployed(reflect.TypeOf(corev1.ConfigMap{}), configMapName.Name)
+	if obj != nil {
+		desired = obj.(*corev1.ConfigMap)
+	}
+
+	if desired == nil {
+
+		desired = &corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ConfigMap",
+				APIVersion: "k8s.io.api.core.v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:         configMapName.Name,
+				GenerateName: "",
+				Namespace:    configMapName.Namespace,
+			},
+		}
+	}
+
+	desired.Data = getJaasManagementPropertiesData(customResource)
+
+	clog.V(1).Info("Requesting configMap for broker JaaS management", "name", configMapName.Name)
+	reconciler.trackDesired(desired)
+
+	clog.V(1).Info("Requesting mount for broker JaaS management config map")
+	return configMapName.Name
+}
+
+func getJaasManagementPropertiesData(customResource *brokerv1beta1.ActiveMQArtemis) map[string]string {
+	security_properties := newPropsWithHeader()
+	fmt.Fprintf(security_properties, "login.config.url.1=file:%v/login.config\n", getJaasManagementPath(customResource))
+	result := map[string]string{
+		"security.properties": security_properties.String(),
+	}
+	login_config := &bytes.Buffer{}
+	fmt.Fprintln(login_config, "management {")
+
+	if jolokia_client.SupportsTokenAuth(customResource.Spec.Version) {
+		fmt.Fprintln(login_config, "  org.apache.activemq.artemis.spi.core.security.jaas.KubernetesLoginModule sufficient")
+		fmt.Fprintln(login_config, "    reload=true")
+		fmt.Fprintln(login_config, "    org.apache.activemq.jaas.kubernetes.role=\"k8s-roles.properties\"")
+		fmt.Fprintf(login_config, "    baseDir=\"%v\"\n", getJaasManagementPath(customResource))
+		fmt.Fprintln(login_config, "  ;")
+		fmt.Fprintln(login_config, "")
+
+		k8s_roles_properties := newPropsWithHeader()
+		bytes, err := sdkk8sutil.GetOperatorNamespace()
+		namespace := "local"
+		if err != nil {
+			clog.V(5).Info("Using --localOnly without --namespace, but unable to determine namespace")
```

Comment:
This is just a placeholder for parsing the `k8s-roles.properties` file when not running on k8s in case the `sdkk8sutil.GetOperatorNamespace()` fails to retrieve the ns. That would happen only if it is running locally and not connected to a k8s cluster but that case is very unlikely to happen so I can just replace it by `""` to avoid misunderstandings.